static/app-strings.json

```diff
@@ -900,6 +900,7 @@
   "Sorry about that. Contact %SITE_HELP_EMAIL% if you continue to have issues.": "Sorry about that. Contact %SITE_HELP_EMAIL% if you continue to have issues.",
```

Comment:
_(not sure if this is legacy or not, can't retry ... need to figure out how to reset account first)_

Scenario:  Account approved for rewards, but haven't claimed "Inviter Bonus" yet
Action:  http://localhost:9090/$/invite/@dev-test:8
Result:  I got the new "go get your Rewards approved" screen :x: .  I also see a bunch of toasts, and saw the inviter bonus.  After refresh, the screen became "you've already claim, we subscribed for you instead".

Maybe just a timing issue, pending fetch or something.